docker-compose.yml

```diff
@@ -38,34 +96,19 @@ services:
       - cms
       - django
 
-  memcached:
-    image: memcached:latest
-    command: ["-m", "1024m"]
-    container_name: core_portal_memcached
-    restart: always
-
-  redis:
-    image: redis:5.0
-    volumes:
-      - core_portal_redis_data:/data
-    container_name: core_portal_redis
-
-  rabbitmq:
-    image: rabbitmq:3.7.17-management
-    volumes:
-      - core_portal_rabbitmq_data:/var/lib/rabbitmq/mnesia/rabbit@core_portal_rabbitmq
-    env_file: ./conf/rabbitmq/rabbitmq.env
-    container_name: core_portal_rabbitmq
-
   websockets:
     image: ${PORTAL_IMAGE}:${PORTAL_TAG}
-    command: 'daphne -b 0.0.0.0 -p 9000 -e ssl:443:privateKey=/etc/ssl/private/portal.key:certKey=/etc/ssl/certs/portal.cer --root-path=/srv/www/portal/server --access-log - --proxy-headers portal.asgi:application'
     volumes:
       - ./conf/certificates/cep.dev.crt:/etc/ssl/certs/portal.cer
       - ./conf/certificates/cep.dev.key/:/etc/ssl/private/portal.key
       - ./conf/portal/settings_secret.py:/srv/www/portal/server/portal/settings/settings_secret.py
       - ./conf/portal/settings_custom.py:/srv/www/portal/server/portal/settings/settings_custom.py
     container_name: core_portal_websockets
+    # ** NOTE: Run this daphne command below to run a production-ready daphne server,
+    # ** matching deployed configurations. Be aware there is no autoreload for local
+    # ** development with this enabled.
+    # command: 'daphne -b 0.0.0.0 -p 9000 -e ssl:443:privateKey=/etc/ssl/private/portal.key:certKey=/etc/ssl/certs/portal.cer --root-path=/srv/www/portal/server --access-log - --proxy-headers portal.asgi:application'
+    command: 'python manage.py runserver 0.0.0.0:9000'
```

Comment:
this is websocket so this command seems out of place.

Reply:
Again, not sure why I added this. Good catch =)

Reply:
Actually, it turns out this is needed by the downstream nginx.

Reply:
We might think we are running daphne on prod, but I do not see that in the configs:

- portal source:     [command: 'python manage.py runserver 0.0.0.0:9000'](https://github.com/TACC/Core-Portal/blob/74430de05db4bff9be37136431fd4ab365c43490/server/conf/docker/docker-compose-dev.all.debug.yml#L100)
- deployment override: https://github.com/TACC/Core-Portal-Deployments/blob/main/frontera-portal/camino/docker-compose.prod.override.yml

I don't see anything overriding the default compose file in the deployments repos.
What am I missing here?